### PR TITLE
Dispatch the PTP ethertype to ptp_print()

### PR DESCRIPTION
**Pull request by fenner**
This implements "IEEE 802.3/Ethernet" mode as described in annex F of the PTP standard.  This uses the same format packets, but directly over Ethernet instead of encapsulated in UDP.

### CODE REVIEW

**Comment by fxlb** [comment]
> This implements "IEEE 802.3/Ethernet" mode as described in annex F of the PTP standard. This uses the same format packets, but directly over Ethernet instead of encapsulated in UDP.

I suggest your comment be added in the commit message body.

File: print-ether.c [hunk 1]
```diff
@@ -100,6 +100,7 @@ const struct tok ethertype_values[] = {
     { ETHERTYPE_GEONET,         "GeoNet"},
     { ETHERTYPE_CALM_FAST,      "CALM FAST"},
     { ETHERTYPE_AOE,            "AoE" },
+    { ETHERTYPE_PTP,            "PTP" },
```
Comment (fxlb) [inline print-ether.c]: Should we print "PTPv2" ?

**Comment by fxlb** [comment]
Please, rebase on top of the-tcpdump-group:master (there were some recent fixes on print-ptp.c).
Also tests/ptp_ethernet.out will require an update.

File: print-ether.c (hunk 1 above)
Comment (fenner) [reply print-ether.c]: IMO, no: the ptp_print() will print PTPv2.  And we don't know that PTPv3 won't use the same ethertype.

**Comment by fenner** [comment]
> > This implements "IEEE 802.3/Ethernet" mode as described in annex F of the PTP standard. This uses the same format packets, but directly over Ethernet instead of encapsulated in UDP.
> 
> I suggest your comment be added in the commit message body.

ok.

File: print-ether.c (hunk 1 above)
Comment (fenner) [reply print-ether.c]: 16:47:07.869101 74:83:ef:01:ac:5b > 01:1b:19:00:00:00, ethertype PTP (0x88f7), length 60: ***PTPv2***, v1 compat : no, msg type : sync msg, length : 44, domain : 0, reserved1 : 0, Flags [two step], NS correction : 0, sub NS correction : 0, reserved2 : 0, clock identity : 0x7483efffff01ac16, port id : 274, seq id : 0, control : 0 (sync msg), log message interval : 0, originTimeStamp : 0 seconds, 0 nanoseconds

Comment (fxlb) [reply print-ether.c]: Agreed.

**Comment by fxlb** [comment]
Thank you!

**Pull request merged**